src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Manifest.component.tsx

```diff
@@ -410,7 +437,48 @@ const ManifestComponent = ({
                     } else {
                         showError(err)
                     }
-                })
+                }).finally(resolve)
+        })
+
+    const uneditedManifest = useMemo(() => {
+        try {
+            const object = YAML.parse(manifest)
+
+            return object?.metadata?.managedFields && hideManagedFields ? getTrimmedManifestData(object) : object
+        } catch (err) {
+            logExceptionToSentry(new Error(`Error: in parsing manifest - ${err.message}`))
+
+            return {}
+        }
+    }, [manifest, hideManagedFields])
+
+    const handleApplyChanges = async () => {
+        setLoading(true)
+        setLoadingMsg('Applying changes')
```

Comment:
can we remove this state and use loading state for message?

Reply:
Would be a little too intrusive. And this file is very risky to edit. 😅